scripts/geopmpy/launcher.py

```diff
@@ -332,6 +332,8 @@ def __init__(self, argv, num_rank=None, num_node=None, cpu_per_rank=None, timeou
             self.config = Config(argv)
             self.is_geopm_enabled = True
             self.is_override_enabled = True
+            if self.config.do_affinity is not None:
+                self.do_affinity = self.config.do_affinity
```

Comment:
I'm not sure what self.do_affinity is for.  I don't see it being used anywhere.  What I do see is self.config.do_affinity.  

Is Launcher.do_affinity dead code?  If yes, the do_affinity checks on line 391 are meaningless.

Reply:
This is a general pattern where the launcher constructor arguments override the configuration opitons if the launcher options are specified.  The config option is then stored in the launcher and it this member is used later to determine if affinity should be done.